proposals/trusted_registry_crd.md

```diff
@@ -0,0 +1,128 @@
+# Introduce a custom resource to represent a (trusted) registry
+
+## TOC
+
+- [Abstract](#abstract)
+- [Problem statement](#problem-statement)
+- [Solution](#solutions)
+- [Use cases](#use-cases)
+  - [Starboard Security Operator supervising trusted registries](#starboard-security-operator-supervising-trusted-registries)
+  - [Repository webhooks integrated with Starboard](#repository-webhooks-integrated-with-starboard)
+
+## Abstract
+
+Common requirements for running secure Kubernetes workloads are:
+
+- Pull container images only from trusted registries and/or repositories. For example, one might allow pulling
+  images from private Harbor repository accessible at https://core.harbor.domain/library/nginx:1.16, but deny pull
+  requests to the core.harbor.domain/library/redis:5 or https://docker.io/library/nginx:1.16 repositories.
```

Comment:
I might be leaping to wrong assumptions here but I think the tag is irrelevant? Suggested changes for clarification (but maybe I have the wrong end of the stick?)

```suggestion
  images from private Harbor repository accessible at https://core.harbor.domain/library/nginx, but deny pull
  requests to https://core.harbor.domain/library/redis (unapproved repo `redis`) or https://docker.io/library/nginx (unapproved registry `docker.io`).
```